### ISSUE TITLE
Shape files stopped working

### ISSUE THREAD

**Issue by GiselleSterry** [nbnuk/nbnatlas-issues#300]
I'm currently trying to download a shapefile from nbn atlas for all the coleoptera records in a specific area. However when I download it, it only has 4 of the necessary files needed to add it to arcGIS. So it only has a DBF, PRJ, SHP and SHX file but I also need a FIX and QIX file in order for it to be read by arcgis. 

Spoken to the user - she has been downloading shapefiles without problem today. Has now stopped working. Tried refresh / on and off etc but not resolved.

**Comment by djtfmartin** [comment]
Shapefiles currently dont include FIX and QIX files. This could be raised as an enhancement.

Ive just downloaded a shapefile with no problem. Can we ask for more details on how its not working ?

**Comment by djtfmartin** [comment]
My bad  - the shapefiles downloads **do** include FIX and QIX files already.

There must be something specific about this user's query thats hitting a problem.

**Comment by CEJohnston** [comment]
More information from the user (received today):

So I had a specific shape file, which was several 2km cells across the UK. I needed all the coleoptera records that were within these 2km squares. I used the analyse tool in NBN atlas to upload the shapefile and get a summary of the data with this area. Because of the 50,000 download limit I did several downloads filtering by licence so I could download all the records and put them into an excel sheet myself. For 3 of the licences it was fine but the final one was missing these 2 files and when I uploaded it to arcGIS nothing showed up.  

Dave - this is the image she sent showing the search she did:
![pastedimage](https://user-images.githubusercontent.com/18594564/28217796-47fa5ebe-68ae-11e7-8eed-a4bbe133b3a7.png)

It looks like the CC0 records are not downloading to shapefile correctly.  Her boundary shapefile that she used for the search is attached.

[AOD_Survey2014_NegativeLocation.zip](https://github.com/nbnuk/nbnatlas-issues/files/1148827/AOD_Survey2014_NegativeLocation.zip)
